testbed/testbed/mock_backend.go

```diff
@@ -172,13 +173,20 @@ func (mb *MockBackend) ConsumeMetricOld(md consumerdata.MetricsData) {
 	}
 }
 
+type TraceDualConsumer interface {
+	consumer.TraceConsumer
+	consumer.TraceConsumerOld
```

Comment:
Why do you need the Old interface?

Reply:
To answer this and the next question, the old interface is used by Prometheus and OpenCensus on the metrics side, and Zipkin on the tracing side.

Reply:
That is no longer true :) for 2 out of 3 :)